### PR TITLE
Update configuration_llama.py: fixed broken link

### LINKED ISSUE #28939: [Broken link] Link not working in LLaMA-2 pretraining_tp doc
The `this document` link on [LLaMA2 doc page](https://huggingface.co/docs/transformers/model_doc/llama2#transformers.LlamaConfig.pretraining_tp) does not work.

### PR DESCRIPTION
**Pull request by AdityaKane2001**
Placed the correct link in `pretraining_tp` docstring in configuration_llama.py

/auto Closes #28939 
/cc @amyeroberts

### CODE REVIEW

File: src/transformers/models/llama/configuration_llama.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ class LlamaConfig(PretrainedConfig):
             End of stream token id.
         pretraining_tp (`int`, *optional*, defaults to 1):
             Experimental feature. Tensor parallelism rank used during pretraining. Please refer to [this
-            document](https://huggingface.co/docs/transformers/parallelism) to understand more about it. This value is
+            document](https://huggingface.co/docs/transformers/main/en/perf_train_gpu_many#tensor-parallelism) to understand more about it. This value is
```
Comment (amyeroberts) [inline src/transformers/models/llama/configuration_llama.py:81]: nit: for the English docs, we don't need to index to `/en/`

```suggestion
            document](https://huggingface.co/docs/transformers/main/perf_train_gpu_many#tensor-parallelism) to understand more about it. This value is
```

**Review by amyeroberts — APPROVED** [review]
Thanks for fixing this! 

Small nit - otherwise we're good to merge!

**Comment by AdityaKane2001** [comment]
@amyeroberts 
Committed change, thanks!

**Comment by AdityaKane2001** [comment]
Thanks Amy!

**Pull request merged**